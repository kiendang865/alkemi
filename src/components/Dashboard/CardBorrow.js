// Sub Component

// Section 1: React low level imports
import React from "react";
import PropTypes from "prop-types";

// Section 2: Material-UI imports
import {
    Box,
    Card,
    CardContent,
    CardHeader,
    Divider,
    Grid,
    Typography,
    makeStyles,
} from "@material-ui/core";
import { fromWei, formatPercent } from "utils/ui";
import { format_str_to_kmb } from "utils/markets";
import Loading from "../Loading/Loading";
// Section 4: Define Styles function taking theme as argument and returning an object
const useStyles = makeStyles((theme) => ({
    root: {
        padding: theme.spacing(3),
        display: "flex",
        alignItems: "center",
        justifyContent: "space-between",
    },
    label: {
        marginLeft: theme.spacing(1),
    },
    item: {
        textAlign: "left",
        justifyContent: "center",
    },
    contentCard: {
        borderBottom: `1px solid ${theme.palette.error.main}`,
    },
}));

function BorrowBalance({ totalBorrow, currency, apr }) {
    const classes = useStyles();
    const [isLoading, setIsLoading] = React.useState(true);
    React.useEffect(() => {
        const timer = setTimeout(() => {
            setIsLoading(false);
        }, 1000);
        return () => clearTimeout(timer);
    }, []);
    // Section 5: Code Component
    return (
        <Card>
            <CardHeader title="Total Borrowed" className={classes.item} />
            <Divider />
            <CardContent className={classes.contentCard}>
                {isLoading ? (
                    <Loading row={1} height={50} />
                ) : (
                    <Box
                        className={classes.item}
                        display="flex"
                        alignItems="center"
                        flexWrap="wrap"
                    >
                        <Grid
                            alignItems="center"
                            container
                            justify="space-between"
                            spacing={3}
                        >
                            <Grid item>
                                <Typography
                                    variant="subtitle1"
                                    color="textSecondary"
                                    align="left"
                                >
                                    Total USD
                                </Typography>
                                <Typography
                                    variant="h3"
                                    color="textPrimary"
                                    align="left"
                                >
                                    ${format_str_to_kmb(fromWei(totalBorrow))}
                                </Typography>
                            </Grid>
                            <Grid item>
                                <Typography
                                    variant="subtitle1"
                                    color="textSecondary"
                                    align="right"
                                >
                                    Borrow APR
                                </Typography>
                                <Typography
                                    variant="h3"
                                    color="textPrimary"
                                    align="right"
                                >
                                    {formatPercent(apr)}
                                </Typography>
                            </Grid>
                        </Grid>
                    </Box>
                )}
            </CardContent>
        </Card>
    );
}

// Section 6: Documentation with PropTypes
BorrowBalance.propTypes = {
    className: PropTypes.string,
};

// Section 7:  Connect styles and export
export default BorrowBalance;
