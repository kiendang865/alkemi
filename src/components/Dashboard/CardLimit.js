// Sub Component

// Section 1: React low level imports
import React from "react";
import PropTypes from "prop-types";

// Section 2: Material-UI imports
import {
    Box,
    Card,
    CardContent,
    LinearProgress,
    Typography,
    makeStyles,
} from "@material-ui/core";

// Section 3: Components & Containers import from the application

// Section 4: Define Styles function taking theme as argument and returning an object
const useStyles = makeStyles((theme) => ({
    root: {
        padding: theme.spacing(3),
        display: "flex",
        alignItems: "right",
        justifyContent: "space-between",
    },
    item: {
        textAlign: "left",
        justifyContent: "center",
    },
    progress: {
        margin: theme.spacing(0, 1, 0, 3),
        flexGrow: 1,
        background: "#414244",
    },
}));

function Limit({ aggregatedBalance, currency, collateralRatio }) {
    const classes = useStyles();
    const data = {
        value: 50,
    };

    // Section 5: Code Component
    return (
        <Card>
            <CardContent className={classes.contentCard}>
                <Typography
                    variant="subtitle1"
                    color="textSecondary"
                    align="left"
                >
                    Borrow Limit
                </Typography>
                <Box display="flex" alignItems="center" flexWrap="wrap">
                    <Typography variant="h3" color="textPrimary">
                        {data.value}%
                    </Typography>
                    <LinearProgress
                        className={classes.progress}
                        value={data.value}
                        color="primary"
                        variant="determinate"
                    />
                </Box>
            </CardContent>
        </Card>
    );
}

// Section 6: Documentation with PropTypes
Limit.propTypes = {
    className: PropTypes.string,
};

// Section 7:  Connect styles and export
export default Limit;
