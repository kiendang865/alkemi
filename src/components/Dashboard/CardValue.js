// Sub Component

// Section 1: React low level imports
import React from "react";
import PropTypes from "prop-types";

// Section 2: Material-UI imports
import {
    Box,
    Card,
    CardContent,
    CardHeader,
    Divider,
    Grid,
    Typography,
    makeStyles,
} from "@material-ui/core";
import { fromWei, formatPercent } from "utils/ui";
import { format_str_to_kmb } from "utils/markets";
import Loading from "../Loading/Loading";
// Section 3: Components & Containers import from the application

// Section 4: Define Styles function taking theme as argument and returning an object
const useStyles = makeStyles((theme) => ({
    root: {
        padding: theme.spacing(3),
        display: "flex",
        alignItems: "right",
        justifyContent: "space-between",
    },
    label: {
        marginLeft: theme.spacing(1),
    },
    item: {
        textAlign: "left",
        justifyContent: "center",
    },
    contentCard: {
        borderBottom: `1px solid ${theme.palette.common.white}`,
    },
}));

function Balance({ aggregatedBalance, currency, collateralRatio }) {
    const classes = useStyles();
    const [isLoading, setIsLoading] = React.useState(true);
    React.useEffect(() => {
        const timer = setTimeout(() => {
            setIsLoading(false);
        }, 1000);
        return () => clearTimeout(timer);
    }, []);
    // Section 5: Code Component
    return (
        <Card>
            <CardHeader title="Account Liquidity" className={classes.header} />
            <Divider />
            <CardContent className={classes.contentCard}>
                {isLoading ? (
                    <Loading row={1} height={50} />
                ) : (
                    <Box
                        className={classes.item}
                        display="flex"
                        alignItems="center"
                        flexWrap="wrap"
                    >
                        <Grid
                            alignItems="center"
                            container
                            justify="space-between"
                            spacing={3}
                        >
                            <Grid item>
                                <Typography
                                    variant="subtitle1"
                                    color="textSecondary"
                                    align="left"
                                >
                                    Total USD
                                </Typography>
                                <Typography
                                    variant="h3"
                                    color="textPrimary"
                                    align="left"
                                >
                                    $
                                    {format_str_to_kmb(
                                        fromWei(aggregatedBalance)
                                    )}
                                </Typography>
                            </Grid>
                            <Grid item>
                                <Typography
                                    variant="subtitle1"
                                    color="textSecondary"
                                    align="right"
                                >
                                    Collateral Ratio
                                </Typography>
                                <Typography
                                    variant="h3"
                                    color="textPrimary"
                                    align="right"
                                >
                                    {formatPercent(collateralRatio)}
                                </Typography>
                            </Grid>
                        </Grid>
                    </Box>
                )}
            </CardContent>
        </Card>
    );
}

// Section 6: Documentation with PropTypes
Balance.propTypes = {
    className: PropTypes.string,
};

// Section 7:  Connect styles and export
export default Balance;
