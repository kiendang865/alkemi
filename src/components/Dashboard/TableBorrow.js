// Sub Component

// Section 1: React low level imports
import React from "react";
import PerfectScrollbar from "react-perfect-scrollbar";

// Section 2: Material-UI imports
import {
    Box,
    fade,
    Table,
    TableBody,
    TableRow,
    Typography,
    makeStyles,
} from "@material-ui/core";

// Section 3: Components & Containers import from the application

//import { useHistory } from "react-router-dom";
import BorrowTableCell from "components/UI/TableCell";
import { formatNumber, formatPercent, calculateEarnings } from "utils/ui";
import Loading from "../Loading/LoadingTable";

// Section 4: Define Styles function taking theme as argument and returning an object
const useStyles = makeStyles((theme) => ({
    root: {
        borderCollapse: "separate",
        borderSpacing: "0 1em",
    },
    startcell: {
        borderRadius: "6px 0px 0px 6px",
    },
    endcell: {
        borderRadius: "0px 6px 6px 0px",
    },
    row: {
        backgroundColor: theme.palette.background.paper,
    },
    imageSize: {
        width: "30px",
        height: "30px",
    },
    label: {
        fontFamily: theme.typography.fontFamily,
        alignItems: "center",
        borderRadius: 2,
        display: "inline-flex",
        flexGrow: 0,
        whiteSpace: "nowrap",
        cursor: "default",
        flexShrink: 0,
        height: 16,
        justifyContent: "center",
        letterSpacing: 0.5,
        minWidth: 20,
        padding: theme.spacing(0.5, 0.5),
        textTransform: "uppercase",
        color: theme.palette.error.main,
        backgroundColor: fade(theme.palette.error.main, 0.1),
    },
}));

export default function BorrowTables({ activeReserves, USDPrice }) {
    const classes = useStyles();

    //  let history = useHistory();

    // Section 5: Code Component
    return (
        <PerfectScrollbar>
            <Table className={classes.root}>
                <TableBody>
                    {!activeReserves.length && (
                        <Loading colSpan={7} row={8} height={63} />
                    )}
                    {activeReserves.map((row) => (
                        <TableRow key={row.unit} className={classes.row}>
                            <BorrowTableCell
                                component="th"
                                scope="Row"
                                className={classes.startcell}
                            >
                                <Box
                                    minWidth={100}
                                    display="flex"
                                    alignItems="center"
                                >
                                    <img
                                        className={classes.imageSize}
                                        src={row.image}
                                        alt={row.symbol}
                                    />
                                    <Box ml={2}>
                                        <Typography
                                            variant="h6"
                                            color="textPrimary"
                                        >
                                            {row.symbol}
                                            {row.unit}
                                        </Typography>
                                        <Typography
                                            variant="body2"
                                            color="textSecondary"
                                        >
                                            {row.name}
                                        </Typography>
                                    </Box>
                                </Box>
                            </BorrowTableCell>
                            <BorrowTableCell align="right">
                                <Typography variant="h6" color="textPrimary">
                                    {formatNumber(row.wallet, row.unit)}
                                </Typography>
                                <Typography
                                    variant="body2"
                                    color="textSecondary"
                                >
                                    Wallet
                                </Typography>
                            </BorrowTableCell>
                            <BorrowTableCell align="right">
                                <Typography variant="h6" color="textPrimary">
                                    {formatNumber(row.balance, row.unit)}
                                </Typography>
                                <Typography
                                    variant="body2"
                                    color="textSecondary"
                                >
                                    Borrowed
                                </Typography>
                            </BorrowTableCell>
                            <BorrowTableCell align="right">
                                <Typography
                                    variant="h6"
                                    color="textPrimary"
                                    className={classes.label}
                                >
                                    {formatPercent(row.apr)}
                                </Typography>
                                <Typography
                                    variant="body2"
                                    color="textSecondary"
                                >
                                    APR
                                </Typography>
                            </BorrowTableCell>
                            <BorrowTableCell
                                align="right"
                                className={classes.endcell}
                            >
                                <Typography variant="h6" color="textPrimary">
                                    {calculateEarnings(
                                        row.balance,
                                        row.principal,
                                        row.unit,
                                        USDPrice
                                    )}
                                </Typography>
                                <Typography
                                    variant="body2"
                                    color="textSecondary"
                                >
                                    EARNINGS
                                </Typography>
                            </BorrowTableCell>
                        </TableRow>
                    ))}
                </TableBody>
            </Table>
        </PerfectScrollbar>
    );
}
