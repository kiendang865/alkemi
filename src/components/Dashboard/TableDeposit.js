// Sub Component

// Section 1: React low level imports
import React from "react";
import PerfectScrollbar from "react-perfect-scrollbar";

// Section 2: Material-UI imports
import {
    Box,
    fade,
    Table,
    TableBody,
    TableRow,
    Typography,
    makeStyles,
} from "@material-ui/core";

//import { useHistory } from "react-router-dom";
import DepositTableCell from "components/UI/TableCell";
import Loading from "../Loading/LoadingTable";
import {
    formatNumber,
    formatPercent,
    //    formatMoney,
    calculateEarnings,
} from "utils/ui";

// Section 4: Define Styles function taking theme as argument and returning an object
const useStyles = makeStyles((theme) => ({
    root: {
        borderCollapse: "separate",
        borderSpacing: "0 1em",
    },
    startcell: {
        borderRadius: "6px 0px 0px 6px",
    },
    endcell: {
        borderRadius: "0px 6px 6px 0px",
    },
    row: {
        backgroundColor: theme.palette.background.paper,
    },
    imageSize: {
        width: "30px",
        height: "30px",
    },
    label: {
        fontFamily: theme.typography.fontFamily,
        alignItems: "center",
        borderRadius: 2,
        display: "inline-flex",
        flexGrow: 0,
        whiteSpace: "nowrap",
        cursor: "default",
        flexShrink: 0,
        height: 16,
        justifyContent: "center",
        letterSpacing: 0.5,
        minWidth: 20,
        padding: theme.spacing(0.5, 0.5),
        textTransform: "uppercase",
        color: theme.palette.success.main,
        backgroundColor: fade(theme.palette.success.main, 0.08),
    },
}));

export default function DepositTables({ activeReserves, USDPrice }) {
    const classes = useStyles();
    //  let history = useHistory();
    // Section 5: Code Component
    return (
        <PerfectScrollbar>
            <Table className={classes.root}>
                <TableBody>
                    {!activeReserves.length && (
                        <Loading colSpan={7} row={8} height={63} />
                    )}
                    {activeReserves.map((row) => (
                        <TableRow key={row.unit} className={classes.row}>
                            <DepositTableCell
                                component="th"
                                scope="row"
                                className={classes.startcell}
                            >
                                <Box
                                    minWidth={100}
                                    display="flex"
                                    alignItems="center"
                                >
                                    <img
                                        className={classes.imageSize}
                                        src={row.image}
                                        alt={row.symbol}
                                    />
                                    <Box ml={2}>
                                        <Typography
                                            variant="h6"
                                            color="textPrimary"
                                        >
                                            {row.symbol}
                                            {row.unit}
                                        </Typography>
                                        <Typography
                                            variant="body2"
                                            color="textSecondary"
                                        >
                                            {row.name}
                                        </Typography>
                                    </Box>
                                </Box>
                            </DepositTableCell>
                            <DepositTableCell align="right">
                                <Typography variant="h6" color="textPrimary">
                                    {formatNumber(row.wallet, row.unit)}
                                </Typography>
                                <Typography
                                    variant="body2"
                                    color="textSecondary"
                                >
                                    Wallet
                                </Typography>
                            </DepositTableCell>
                            <DepositTableCell align="right">
                                <Typography variant="h6" color="textPrimary">
                                    {formatNumber(row.balance, row.unit)}
                                </Typography>
                                <Typography
                                    variant="body2"
                                    color="textSecondary"
                                >
                                    Supplied
                                </Typography>
                            </DepositTableCell>
                            <DepositTableCell align="right">
                                <Typography
                                    variant="h6"
                                    className={classes.label}
                                >
                                    {formatPercent(row.apr)}
                                </Typography>
                                <Typography
                                    variant="body2"
                                    color="textSecondary"
                                >
                                    APR
                                </Typography>
                            </DepositTableCell>
                            <DepositTableCell
                                align="right"
                                className={classes.endcell}
                            >
                                <Typography variant="h6" color="textPrimary">
                                    {calculateEarnings(
                                        row.balance,
                                        row.principal,
                                        row.unit,
                                        USDPrice
                                    )}
                                </Typography>
                                <Typography
                                    variant="body2"
                                    color="textSecondary"
                                >
                                    Earnings
                                </Typography>
                            </DepositTableCell>
                        </TableRow>
                    ))}
                </TableBody>
            </Table>
        </PerfectScrollbar>
    );
}
