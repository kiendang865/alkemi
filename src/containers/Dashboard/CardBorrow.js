// Section 1: React/Redux low level imports
import React from "react";
import { useSelector } from "react-redux";
import { compose } from "redux";

// Section 2: internal imports
import withDrizzle from "containers/Drizzle/withDrizzle";
import CardBorrow from "components/Dashboard/CardBorrow";

// Section 3: Selectors / hooks
import { getTotalBorrow } from "redux/selectors/totalBalance";
import { getAggregatedBorrowAPR } from "redux/selectors/data/borrow";
import useFetchPrice from "hooks/useFetchPrice";

// Section 4: Code Component
const CardValueContainer = () => {
    useFetchPrice(); // gets current USD price

    const totalBorrow = useSelector(getTotalBorrow);

    const totalBorrowAPR = useSelector(getAggregatedBorrowAPR); // combined portfolio APR rate

    return (
        <CardBorrow
            totalBorrow={totalBorrow}
            currency="USD"
            difference={0}
            apr={totalBorrowAPR}
        />
    );
};

// const mapStateToProps = (state) => ({
//     activeReserves: getBorrowActive(state),
// });

export default compose(
    withDrizzle()
    //  connect(mapStateToProps)
)(CardValueContainer);
