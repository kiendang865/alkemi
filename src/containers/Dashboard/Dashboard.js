import React from "react";
import Dashboard from "components/Dashboard/Dashboard";

const DashboardContainer = () => {
    return <Dashboard />;
};

export default DashboardContainer;
