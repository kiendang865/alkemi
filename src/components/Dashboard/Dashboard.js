// Main Component

// Section 1: React low level imports
import React, { useState } from "react";

// Section 2: Material-UI imports
import {
    Box,
    Container,
    Divider,
    Grid,
    makeStyles,
    Tab,
    Tabs,
} from "@material-ui/core";

// Section 3: Components & Containers import from the application
import DashCardValue from "containers/Dashboard/CardValue";
import DashCardDeposit from "containers/Dashboard/CardDeposit";
import DashCardBorrow from "containers/Dashboard/CardBorrow";
import TableDeposit from "containers/Dashboard/TableDeposit";
import TableBorrow from "containers/Dashboard/TableBorrow";

// Section 4: Define Styles function taking theme as argument and returning an object
const useStyles = makeStyles((theme) => ({
    root: {
        backgroundColor: theme.palette.background.dark,
        minHeight: "100%",
        paddingTop: theme.spacing(3),
        paddingBottom: theme.spacing(3),
    },
    container: {
        [theme.breakpoints.up("lg")]: {
            paddingLeft: 0,
            paddingRight: 0,
        },
    },
}));

const tabs = [
    { value: "tabledeposit", label: "Active Deposits" },
    { value: "tableborrow", label: "Active Borrowed" },
];

function Dashboard() {
    const classes = useStyles();
    const [currentTab, setCurrentTab] = useState("tabledeposit");
    const handleTabsChange = (event, value) => {
        setCurrentTab(value);
    };

    // Section 5: Code Component
    return (
        <Container maxWidth={false} className={classes.container}>
            <Grid container spacing={3}>
                <Grid item lg={4} sm={6} xs={12}>
                    <DashCardValue />
                </Grid>
                <Grid item lg={4} sm={6} xs={12}>
                    <DashCardDeposit />
                </Grid>
                <Grid item lg={4} sm={6} xs={12}>
                    <DashCardBorrow />
                </Grid>
                <Grid item lg={12} sm={12} xs={12}>
                    <Box>
                        <Box mt={2}>
                            <Tabs
                                onChange={handleTabsChange}
                                scrollButtons="auto"
                                value={currentTab}
                                textColor="secondary"
                                variant="scrollable"
                            >
                                {tabs.map((tab) => (
                                    <Tab
                                        key={tab.value}
                                        label={tab.label}
                                        value={tab.value}
                                    />
                                ))}
                            </Tabs>
                        </Box>
                        <Divider />
                        <Box py={3} pb={6}>
                            {currentTab === "tabledeposit" && <TableDeposit />}
                            {currentTab === "tableborrow" && <TableBorrow />}
                        </Box>
                    </Box>
                </Grid>
            </Grid>
        </Container>
    );
}

// Section 6: Documentation with PropTypes

// Section 7:  Connect styles and export
export default Dashboard;
