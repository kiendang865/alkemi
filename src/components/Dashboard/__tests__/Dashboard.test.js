import React from "react";
import toJson from "enzyme-to-json";

import Dashboard from "components/Dashboard/Dashboard";
import createShallow from "utils/tests-utils/createShallow";

describe("<Dashboard />", () => {
    let shallow;
    let wrapper;

    beforeAll(() => {
        shallow = createShallow({ dive: 1 });
    });
    beforeEach(() => {
        wrapper = shallow(<Dashboard />);
    });

    it("matches snapshot", () => {
        expect(toJson(wrapper)).toMatchSnapshot();
    });
});
