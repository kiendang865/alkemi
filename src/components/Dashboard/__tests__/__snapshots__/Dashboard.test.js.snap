// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Dashboard /> matches snapshot 1`] = `
<ForwardRef(Container)
  className="makeStyles-container-2"
  classes={
    Object {
      "disableGutters": "MuiContainer-disableGutters",
      "fixed": "MuiContainer-fixed",
      "maxWidthLg": "MuiContainer-maxWidthLg",
      "maxWidthMd": "MuiContainer-maxWidthMd",
      "maxWidthSm": "MuiContainer-maxWidthSm",
      "maxWidthXl": "MuiContainer-maxWidthXl",
      "maxWidthXs": "MuiContainer-maxWidthXs",
      "root": "MuiContainer-root",
    }
  }
  maxWidth={false}
>
  <WithStyles(ForwardRef(Grid))
    container={true}
    spacing={3}
  >
    <WithStyles(ForwardRef(Grid))
      item={true}
      lg={4}
      sm={6}
      xs={12}
    >
      <CardValueContainer />
    </WithStyles(ForwardRef(Grid))>
    <WithStyles(ForwardRef(Grid))
      item={true}
      lg={4}
      sm={6}
      xs={12}
    >
      <DrizzledComponent />
    </WithStyles(ForwardRef(Grid))>
    <WithStyles(ForwardRef(Grid))
      item={true}
      lg={4}
      sm={6}
      xs={12}
    >
      <DrizzledComponent />
    </WithStyles(ForwardRef(Grid))>
    <WithStyles(ForwardRef(Grid))
      item={true}
      lg={12}
      sm={12}
      xs={12}
    >
      <Styled(MuiBox)>
        <Styled(MuiBox)
          mt={2}
        >
          <WithStyles(ForwardRef(Tabs))
            onChange={[Function]}
            scrollButtons="auto"
            textColor="secondary"
            value="tabledeposit"
            variant="scrollable"
          >
            <WithStyles(ForwardRef(Tab))
              key="tabledeposit"
              label="Active Deposits"
              value="tabledeposit"
            />
            <WithStyles(ForwardRef(Tab))
              key="tableborrow"
              label="Active Borrowed"
              value="tableborrow"
            />
          </WithStyles(ForwardRef(Tabs))>
        </Styled(MuiBox)>
        <WithStyles(ForwardRef(Divider)) />
        <Styled(MuiBox)
          pb={6}
          py={3}
        >
          <DrizzledComponent />
        </Styled(MuiBox)>
      </Styled(MuiBox)>
    </WithStyles(ForwardRef(Grid))>
  </WithStyles(ForwardRef(Grid))>
</ForwardRef(Container)>
`;
